src/app/lib/utilities/getRemoteConfig/getRemoteConfigWithCache.js

```diff
@@ -0,0 +1,14 @@
+import Cache from 'lru-cache';
+import getRemoteConfig from '.';
+
+const cacheMaxItems = parseInt(process.env.SIMORGH_CONFIG_CACHE_ITEMS, 10);
```

Comment:
Currently our ESLint rule requires the radix for `parseInt` be passed it https://eslint.org/docs/rules/radix
It's not the default value, but rather part of the parser. The default value is set to 100 entries at the moment.

The value needs to be parsed as well because environment variables are always strings.